### PR TITLE
Customer view bookings

### PR DESCRIPTION
**Pull request by phucynwa**

### CODE REVIEW

**Comment by phucynwa** [comment]
https://dev.framgia.com/issues/184352

**Comment by phucynwa** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -176,3 +176,171 @@ textarea {
...
+  color: $lighter-pink;
+  margin: 5px;
+}
+
+.room-info img {
+  width: 100%;
+}
+
+.room-info {
+ position:relative;
```
Comment (mdmahmudurrahman) [inline app/assets/stylesheets/custom.scss]: 1. Indentation
2. Add a white space between key and the value.

File: app/views/bookings/new.html.erb [hunk 2]
```diff
@@ -6,16 +6,19 @@
   <div class="col-md-6 contact-form">
     <h3><%= raw t(".create_booking_title") %></h3>
     <%= form_for @booking do |f| %>
-      <%= f.label :rooms %>
-      <select name="rooms[]" multiple>
-        <%= all_room.each_with_index do |room, i| %>
-          <option value="<%= room.id %>"><%= room.label %></option>
+      <%= f.label :rooms %><br>
+      <div class="row">
+        <% all_room.where(status: :free).each do |room| %>
```
Comment (mdmahmudurrahman) [inline app/views/bookings/new.html.erb]: Use scope for it

**Comment by mdmahmudurrahman** [comment]
Commented

**Comment by phucynwa** [comment]
ready

File: app/models/room.rb [hunk 3]
```diff
@@ -11,4 +11,5 @@ class Room < ApplicationRecord
 
   scope :by_category_id, ->(category_id){where category_id: category_id if category_id.present?}
   scope :by_floor, ->(floor){where floor: floor if floor.present?}
+  scope :free, ->{where status: :free}
```
Comment (huongnguyenmta) [inline app/models/room.rb]: Viết như này để sau dùng lại nhé em:
```ruby
scope :by_status, ->(status){where status: status}
```

File: app/views/bookings/_booking.html.erb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<div class="blog-list">
+  <h4><%= t ".booked_at", time: booking.updated_at.strftime("%H:%M %d/%m/%Y") %></h4>
+  <ul>
+    <li><a>
+      <i class="fa fa-caret-right"></i>
+      <%= t ".number_rooms", count: booking.rooms.count %>
+    </a></li>
+    <li><a>
+      <i class="fa fa-caret-right"></i>
+      <%= t ".start_time", time: booking.start_time.strftime("%H:%M %d/%m/%Y") %>
```
Comment (huongnguyenmta) [inline app/views/bookings/_booking.html.erb]: Dùng `I18n format time`

**Comment by huongnguyenmta** [comment]
commented

File: app/views/bookings/_booking.html.erb (hunk 4 above)
Comment (devtuananh) [reply app/views/bookings/_booking.html.erb]: <%= t ".start_time", time: l(booking.start_time, format: :own) %>

**Comment by mdmahmudurrahman** [comment]
commented

**Comment by phucynwa** [comment]
ready

**Pull request merged**